.circleci/scripts/binary_linux_test.sh

```diff
@@ -52,13 +52,6 @@ if [[ "\$python_nodot" = *39* ]]; then
   NUMPY_PIN=">=1.20"
 fi
 
-if [[ "\$python_nodot" = *38* ]]; then
-  # sympy 1.12.1 is the last version that supports Python 3.8
-  SYMPY_PIN="==1.12.1"
```

Comment:
You could search and replace the occurance of `1.2.1` and `1.3.0`.

Reply:
I only want to change the conda ones though, I do want pypi to be on these newer versions